src/js/fs.js

```diff
@@ -263,26 +255,25 @@ fs.readFileSync = function(path) {
   return Buffer.concat(buffers);
 };
 
+fs.writeFile = function(path, outdata, callback) {
+  var data = ensureBuffer(outdata);
```

Comment:
The functions like this implicitly allow every type of args allowed in `Buffer` constructor. I don't agree that. Plus, when the existing API is changed, corresponding document and test case are also considered. I also want more types are supported for `outdata` though.